client/src/hooks/helpers/useAvailableFundsEth.ts

```diff
@@ -14,7 +14,6 @@ export default function useAvailableFundsEth(): UseBalanceReturnType<{
   const { address } = useAccount();
   const { data: blockNumber } = useBlockNumber({
     chainId: networkConfig.id,
-    watch: true,
```

Comment:
This was resulting in fetching on each block, even when user left the tab open in the background. Biggest chunk of operations in our Alchemy is `eth_blockNumber`.